### PR TITLE
Mobile project builder noah

### PR DESCRIPTION
**Pull request by noahmalmed**
Staging branch URL:

Fixes # .

Describe your changes.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/pages/lab/mobile/mobile_previewer.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+const TealBox = ({text}) => {
+    return (
+        <div style={styles.tealBoxContainer}>
+            <span style={{color: 'white'}}> {text} </span>
+        </div>
+    )
+}
+
+const styles = {
```
Comment (wgranger) [inline app/pages/lab/mobile/mobile_previewer.jsx:83]: Hard to break habits. This works alright for now, but if merged into PFE master at some point, these styles would need to be converted to classes.

Comment (noahmalmed) [reply app/pages/lab/mobile/mobile_previewer.jsx:83]: 🖕

File: app/pages/lab/mobile/mobile_previewer.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { zooTheme } from '../../../theme'
+
+export default class MobilePreviewer extends Component {
+  constructor(props) {
+    super(props);
```
Comment (wgranger) [inline app/pages/lab/mobile/mobile_previewer.jsx:7]: `super(props)` -> the "props" argument isn't necessary unless props are being manipulated in the constructor.

Comment (noahmalmed) [reply app/pages/lab/mobile/mobile_previewer.jsx:7]: I could probably just get rid of the constructor. I don't use it

Comment (noahmalmed) [reply app/pages/lab/mobile/mobile_previewer.jsx:7]: 🖕

**Pull request merged**